configure.sh

```diff
@@ -43,6 +43,8 @@ function platform() {
 		PLATFORM='linux'
 	elif [[ "$unamestr" == 'Darwin' ]]; then
 		PLATFORM='mac'
+	elif [[ "$unamestr" =~ 'MINGW' ]]; then
+		PLATFORM='windows'
 	fi
```

Comment:
How about adding an `else` and exiting with an error if the platform is not recognized?

Reply:
Done